e2e-tests/test/admin/customer/list_pagination.test.ts

```
@@ -0,0 +1,47 @@
+/**
+ * Issue #1398: 管理画面の会員一覧で LIMIT が SQL に反映されず
+ * 全件取得される回帰 (PR #1116 起因) の E2E 回帰テスト.
+ *
+ * 前提: CI/ローカル環境で `--customers=30` 以上の会員フィクスチャが投入されていること.
+ */
+test.describe('会員一覧のページネーション (Issue #1398)', () => {
+
+  /**
+   * 会員レコードの先頭行 (rowspan="2" を持つ td を含む tr) から customer_id を取得.
+   */
+  async function collectCustomerIds(page: Page): Promise<string[]> {
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)